src/navigators/AuthStack.js

```diff
@@ -10,8 +10,8 @@ const Stack = createStackNavigator();
 
 const AuthStack = () => (
   <Stack.Navigator>
-    <Stack.Screen name={LOGIN_SCREEN} component={LoginScreen} />
-    <Stack.Screen name={SIGN_UP_SCREEN} component={SignUpScreen} />
+    <Stack.Screen name={LOGIN_SCREEN} component={LoginScreen} options={{ headerShown: false }} />
```

Comment:
What do you think about hiding the header gloabaly. It seems that is more common to not use the header than to use it